pimpmycause/pimpmycause/templates/registration/registration_complete.html

```diff
@@ -3,8 +3,8 @@
 {% load static %}
 
 {% block main %} 
-	<div class="pnl-title page-register">
-		<h1>Thanks!</h1>
+	<div class="page-register pnl">
+		<h1 class="pnl-title">Thanks!</h1>
```

Comment:
is the reason of you adding class `pnl` to all pages to then be able to create BEM `pnl-title` here?